tools/unused.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+import os
+import re
+import b
+
+# @run_on_docker
+def register(command):
+    # Install help
+    command.help = "Creates a list of unused or commented out modules"
+
+
+# @run_on_docker
+def run(**kwargs):
+
+    source_path = b.cmake_cache["NUbots_SOURCE_DIR"]
+    roles_path = source_path + "/roles"
+    modules_path = source_path + "/modules"
```

Comment:
This one is in the cmake_cache already
```suggestion
    modules_path = os.path.join(source_path, b.cmake_cache['NUCLEAR_MODULE_DIR'])
```